.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
Can you change this to the version used elsewhere? ([this](https://github.com/AtomLinter/linter-pycodestyle/blob/2b90bb8a1648b45eefa1d74db0325822c0c140f2/.editorconfig)).

Reply:
Sure no problem 👍